index.html

```diff
@@ -9,6 +9,32 @@
 </head>
 
 <body>
+    <div class="container">
+        <header>
+            <nav class="navigation">
+                <div class="logo-wrap">
+                    <a href="index.html"><img src="images/logo 1.svg" alt="logo"></a>
+                </div>
+                <div class="navigation-header">
+                    <a href="">Home</a>
+                    <a href="">About</a>
+                    <a href="">Tech Atack</a>
```

Comment:
Erro de digitação
```suggestion
                    <a href="">Tech Attack</a>
```

Qual é a idéia dessa página?